test/output_test.dart

```diff
@@ -59,4 +59,34 @@ void main() {
       });
     });
   });
+
+  // Regression test for sass/dart-sass#417.
+  group("preserve trailing loud comments", () {
+    // No need for "in Sass" cases as it's not possible to have
+    // trailing loud comments in the Sass syntax.
+    group("in SCSS", () {
```

Comment:
I'd just fold this into the previous group name to save on some indentation.

Reply:
done